src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -42,7 +48,13 @@
     private MenuItem helpMenuItem;
 
     @FXML
-    private StackPane personListPanelPlaceholder;
+    private StackPane travelPlannerPanelPlaceholder;
```

Comment:
Is it better to have ScrollPane here? As we might have lots of TravelPlan and the Pane cant show everything

Reply:
Just tested, the ListView implemented in TravelPlannerPanel implements scrolling already!